scalalib/src/mill/scalalib/PublishModule.scala

```diff
@@ -247,23 +247,27 @@ trait PublishModule extends JavaModule { outer =>
   /**
    * Publish artifacts to a local Maven repository.
    * @param m2RepoPath The path to the local repository  as string (default: `$HOME/.m2repository`).
+   *                   If not set, falls back to `maven.repo.local` system property or `~/.m2/repository`
    * @return [[PathRef]]s to published files.
    */
-  def publishM2Local(m2RepoPath: String = (os.home / ".m2/repository").toString())
-      : Command[Seq[PathRef]] = Task.Command {
-    publishM2LocalTask(Task.Anon {
-      os.Path(m2RepoPath, T.workspace)
-    })()
-  }
+  def publishM2Local(m2RepoPath: String = getM2LocalRepoPath.toString()): Command[Seq[PathRef]] =
```

Comment:
This doesn't look right. Since `getM2LocalRepoPath` is a task, the `toString` return it's task-name instead of the actual path.

Reply:
Oh, this is already merged. Either I read the code wrong or we have no proper test coverage, otherwise we should see some test failures.